packages/fxa-content-server/app/scripts/templates/pair/failure.mustache

```diff
@@ -1,18 +1,18 @@
-<div class="card">
-  <header>
-    <h1 id="fxa-pair-failure-header" class="card-header">
-      {{#t}}Pairing not successful{{/t}}
-    </h1>
-  </header>
-
+<div class="card-base">
   <section>
     <div class="error"></div>
     <div class="success"></div>
 
     <form novalidate>
       <div class="bg-image-pair-fail"></div>
 
-      <p>{{#t}}The setup process was terminated. Please sign in with your email.{{/t}}</p>
+      <header>
+        <h1 id="fxa-pair-failure-header" class="card-header">
+          {{#t}}Device pairing failed.{{/t}}
```

Comment:
I'll remove the period on this with my next push.

Also just noticed, `The setup couldn't be completed.` should use a curly quote so I'll get that too.